src/specs.md

```diff
@@ -13,6 +13,41 @@ GNU Affero General Public License for more details.
 You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+```act
+behaviour wards of MkrAuthority
+interface wards(address usr)
+
+for all
+
```

Comment:
Let's be consistent with spacing...I prefer the more compact version, personally:
```
for all
    May : uint256
```

Reply:
fixed!